app/assets/stylesheets/passengers.scss

```diff
@@ -16,7 +16,19 @@ p, ol, ul, td {
   font-size: 13px;
   line-height: 18px;
 }
-
+strong.bold_and_underline {
```

Comment:
Maybe think of renaming `.bold_and_underline` to something else. For the same reason as @Anbranin's comment below